build.gradle

```diff
@@ -86,13 +85,13 @@ uploadArchives {
 
                 developers {
                     developer {
-                        id = 'Ridhwana'
-                        name = 'Ridhwana Khan'
-                        email = 'ridhwana@smileidentity.com'
+                        id = 'Arlon'
```

Comment:
Let's put support details here, including support@smileidentity.com this is not critical to the deploy but is metadata that people can pick